tests/test_scanner.py

```diff
@@ -590,6 +602,7 @@ def test_scan_directory_path():
             Global("__builtin__", "dict", SafetyLevel.Suspicious),
             Global("__builtin__", "apply", SafetyLevel.Dangerous),
             Global("__builtin__", "getattr", SafetyLevel.Dangerous),
+            Global("__builtin__", "getattr", SafetyLevel.Dangerous),
```

Comment:
Is this because we're using pickle 2 ? Could be interesting to have a test with pickle v4

Reply:
The test actually covers both v2 and v4 -- each contributing one `Dangerous` entry to the list from what I saw.
```
    initialize_pickle_file(f"{_root_path}/data/malicious15a.pkl", Malicious15(), 2)
    initialize_pickle_file(f"{_root_path}/data/malicious15b.pkl", Malicious15(), 4)
```